hubgrep/frontend_blueprint/templates/components/search_form/search_form.html

```diff
@@ -45,28 +45,28 @@
             </div>
 
             <div class="misc">
-                <div class="misc-forks">
+                <div class="filter-item misc-forks" title="{{ constants.TOOLTIP_FILTER_EXCLUDE_FORK }}">
                     <input type="checkbox" id="forks" name="{{ constants.FORM_ARGS.exclude_forks }}"
                            {% if form.exclude_forks %}checked{% endif %}>
-                    <label for="forks" title="{{ constants.TOOLTIP_FILTER_EXCLUDE_FORK }}">
+                    <label for="forks">
                         {{ gettext("exclude forks") | safe }}</label>
                 </div>
-                <div class="misc-archived">
+                <div class="filter-item misc-archived" title="{{ constants.TOOLTIP_FILTER_EXCLUDE_ARCHIVED }}">
                     <input type="checkbox" id="archived" name="{{ constants.FORM_ARGS.exclude_archived }}"
                            {% if form.exclude_archived %}checked{% endif %}>
-                    <label for="archived" title="{{ constants.TOOLTIP_FILTER_EXCLUDE_ARCHIVED }}">
+                    <label for="archived">
                         {{ gettext("exclude archived") | safe }}</label>
                 </div>
-                <div class="misc-empty">
+                <div class="filter-item misc-empty" title="{{ constants.TOOLTIP_FILTER_EXCLUDE_EMPTY }}">
                     <input type="checkbox" id="empty" name="{{ constants.FORM_ARGS.exclude_empty }}"
                            {% if form.exclude_empty %}checked{% endif %}>
-                    <label for="empty" title="{{ constants.TOOLTIP_FILTER_EXCLUDE_EMPTY }}">
+                    <label for="empty">
                         {{ gettext("exclude empty") | safe }}</label>
                 </div>
-                <div class="misc-mirror">
+                <div class="filter-item misc-mirror" title="{{ constants.TOOLTIP_FILTER_EXCLUDE_MIRROR }}">
                     <input type="checkbox" id="mirror" name="{{ constants.FORM_ARGS.exclude_mirror }}"
                            {% if form.exclude_mirror %}checked{% endif %}>
-                    <label for="mirror" title="{{ constants.TOOLTIP_FILTER_EXCLUDE_MIRROR }}">
+                    <label for="mirror">
```

Comment:
move title to cover the whole item, not just partially, also give the item a class so I can target these specific elements